<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Serial #</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Run #</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blaster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="branButt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Brandon</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="patButt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pat</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="evButt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Evan</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="bagNumBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="runNumText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="runWeight">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="recentRun">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Most Recent Run:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
